Formula/eye-d3.rb

```diff
@@ -3,15 +3,14 @@ class EyeD3 < Formula
 
   desc "Work with ID3 metadata in .mp3 files"
   homepage "https://eyed3.nicfit.net/"
-  url "https://eyed3.nicfit.net/releases/eyeD3-0.9.5.tar.gz"
-  mirror "https://files.pythonhosted.org/packages/3a/7a/07fc7a0e4f7913f599dae950ea5024f006ccef2bc1bbffba288ed8fdfcab/eyeD3-0.9.5.tar.gz"
-  sha256 "faf5806197f2093e82c2830d41f2378f07b3a9da07a16fafb14fc6fbdebac50a"
-  license "GPL-3.0"
-  revision 2
+  url "https://eyed3.nicfit.net/releases/eyeD3-0.9.6.tar.gz"
+  mirror "https://files.pythonhosted.org/packages/3a/7a/07fc7a0e4f7913f599dae950ea5024f006ccef2bc1bbffba288ed8fdfcab/eyeD3-0.9.6.tar.gz"
+  sha256 "4b5064ec0fb3999294cca0020d4a27ffe4f29149e8292fdf7b2de9b9cabb7518"
+  license "GPL-3.0-or-later"
 
   livecheck do
-    url "https://eyed3.nicfit.net/releases/"
-    regex(/href=.*?eyeD3[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    url "https://github.com/nicfit/eyeD3.git"
+    strategy :github_latest
```

Comment:
```suggestion
    url :stable
```

`:stable` check is sufficient

Reply:
It was using `:stable` previously, but failed audit because `https://eyed3.nicfit.net/releases/` seems to have restricted access.

Reply:
This is what happens when I replace it with `:stable`:
```
❯ brew livecheck --debug eye-d3
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/eye-d3.rb

Formula:          eye-d3
Livecheckable?:   Yes

URL:              https://eyed3.nicfit.net/releases/eyeD3-0.9.6.tar.gz
Strategy:         None
Error: eye-d3: Unable to get versions
```

Reply:
I see, switch url and mirror would do the job